integrated_channels/api/v1/sap_success_factors/views.py

```diff
@@ -15,3 +18,59 @@ class SAPSuccessFactorsConfigurationViewSet(PermissionRequiredForIntegratedChann
     permission_required = 'enterprise.can_access_admin_dashboard'
 
     configuration_model = SAPSuccessFactorsEnterpriseCustomerConfiguration
+
+
+@api_view(['GET'])
+@permission_classes([permissions.IsAuthenticated])
+def retrieve_additional_userinfo(request):
+    """
+    Retrieve additional user information from SAP SuccessFactors.
+    
+    GET params:
+        org_id: Organization ID in SAP
```

Comment:
This value actually represents the organization ID in Auth0. The intended flow is to first fetch the [EnterpriseCustomer](https://github.com/openedx/edx-enterprise/blob/47f37480285e8c962d90556c9f71ad877d1c510b/enterprise/models.py#L174) using this org_id as "auth_org_id", and then retrieve the corresponding SAP SuccessFactors configuration for that customer. From there, we can extract the company_id:
https://github.com/openedx/edx-enterprise/blob/47f37480285e8c962d90556c9f71ad877d1c510b/integrated_channels/sap_success_factors/models.py#L90